view/frontend/templates/instant/wrapper.phtml

```diff
@@ -23,44 +24,26 @@
                     <div class="col-md-12">
                         <div>
                             {{#second_bar}}
-                            <div id="instant-search-bar-container">
-                                <div id="instant-search-box">
-                                    <div class="instant-search-bar-label">
-                                        <span><?php echo $block->escapeHtml(__('Current search')); ?></span>
-                                    </div>
-                                    <div class="instant-search-bar-wrapper">
-                                        <label for="instant-search-bar">
-                                            <?php echo $block->escapeHtml(__('Search :')); ?>
-                                        </label>
-
-                                        <input placeholder="<?php echo $block->escapeHtml(__('Search for products')) ?>"
-                                               id="instant-search-bar"
-                                               type="text"
-                                               autocomplete="off"
-                                               spellcheck="false"
-                                               autocorrect="off"
-                                               autocapitalize="off" />
-                                    </div>
-                                </div>
-                            </div>
+                               <div id="instant-search-bar"></div>
                             {{/second_bar}}
                         </div>
                     </div>
                 </div>
                 <div class="row">
-                    <div>
+                    <div class="col-md-12">
                         <div itemscope itemtype="http://schema.org/ItemList" class="hits">
-                            <div class="infos">
+                            <div class="algolia-infos">
```

Comment:
renamed this to stay consistent with algolia-banner.
There is probably more renaming we can do to make it clear which ids and classes we own.
Open to suggestions

Reply:
infos -> info (always singular in English)